examples/lighting-app/telink/Readme.md

```diff
@@ -0,0 +1,180 @@
+![Telink B91 EVK](http://wiki.telink-semi.cn/wiki/assets/Hardware/B91_Generic_Starter_Kit_Hardware_Guide/connection_chart.png)
+
+## Build
```

Comment:
can it be built from vscode (can we include the SDK inside the vscode docker image)?
Developers are much more likely to test from there rather than checking out a separate docker pull. Also when breakages happen, I would like to debug locally.

Reply:
@andy31415 Yes, it is possible to build it with vscode. I've add specific task for that in .vscode/tasks.json.